pkg/block/block.go

```diff
@@ -93,6 +96,7 @@ const (
 	STORAGE_CONTROLLER_NVME                      // Non-volatile Memory Express
 	STORAGE_CONTROLLER_VIRTIO                    // Virtualized storage controller/driver
 	STORAGE_CONTROLLER_MMC                       // Multi-media controller (used for mobile phone storage devices)
+	STORAGE_CONTROLLER_VIRTUAL                   // virtual controller, i.e. for a loop device
```

Comment:
Sorry @Itxaka I didn't make myself clear in the original review... I thought `STORAGE_CONTROLLER_LOOP` was good. It's only the `DRIVE_TYPE_LOOP` that I wanted to change to `DRIVE_TYPE_VIRTUAL`... Can we swtich the storage controller back please? :)

Reply:
done!